### DISCUSSION TITLE
Hay que hacer un README.md para cada kata?

### DISCUSSION THREAD

**Discussion by JhosianGtz** [LaunchX-InnovaccionVirtual/CursoIntroPython#65] [Q&A] [answered]
Tenemos que hacer un README.md para cada kata con capturas?

**Comment by Texhnolyze47** [comment]
No es necesario

**Comment by MSuarez0** [comment, marked as answer]
lo acaban de mencionar, de que no es necesario pero es a tu gusto, y es valido 👍

**Comment by Dfredude** [comment]
Los notebooks ya son lo suficientemente descriptivos para no necesitarlo, lo cual agregar un README.md sería como echarle agua al mar.

Tampoco es un requerimiento para el curso. Saludos

**Comment by doguedogue** [comment]
Siempre es buena una explicación introductoria de un repositorio, indicar de que se trata y que hay dentro (objetivo del README.md).

Dijeron que era opcional, así que como gustes. 

No olvides marcar como RESPONDIDA tu pregunta si así lo consideras.

![image](https://user-images.githubusercontent.com/23409026/153455590-6c36e572-ed9f-4fb4-925c-5a2a4a445084.png)

**Comment by JhosianGtz** [comment]
Gracias por las respuestas c:
